collections-arrays/ConvertHexStringToByteArray/src/ConvertHexStringToByteArrayBenchmark/ConvertHexStringToByteArrayBenchmarks.cs

```diff
@@ -0,0 +1,64 @@
+﻿using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Order;
+using ConvertHexStringToByteArray.Library;
+
+namespace ConvertHexStringToByteArray.Benchmarks;
+
+[Orderer(SummaryOrderPolicy.FastestToSlowest)]
+[GroupBenchmarksBy(BenchmarkLogicalGroupRule.ByParams)]
+public class ConvertHexStringToByteArrayBenchmarks
+{
+    private static readonly int[] ByteArrayLengths =
+    {
+        32,
+        128,
+        4096,
+        1_048_576
+    };
+
+    private readonly List<string> _sourceData = new();
+
+    public ConvertHexStringToByteArrayBenchmarks()
+    {
+        var random = new Random(42);
```

Comment:
@AndreCarvalho - That is the source 😅, but also so that the benchmark is consistent between all runs and for all functions. Using a seed means each run produces the same "random" output.